ManageTeamsIdentityMobileAndDesktop/README.md

```diff
@@ -0,0 +1,50 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-communication-services
+---
+
+# Create and manage Communication access tokens for Teams users in mobile and desktop applications
+
+This code sample walks you through the process of acquiring a Communication Token Credential by exchanging an Azure AD token of a user with a Teams license for a valid Communication access token.
```

Comment:
> Communication Token Credential

not true in the case of this package. we're just acquiring a token. let's keep it this way and improve the samples once we release GA. (similarly to https://github.com/Azure-Samples/communication-services-javascript-quickstarts/pull/73#discussion_r921096545)

Reply:
ok, for all platforms I will create one work item for the doc & samples improvement